tests/probes/yamlfilecontent/test_probes_yamlfilecontent_offline_mode.sh

```diff
@@ -15,14 +15,23 @@ function test_probes_yamlfilecontent_offline_mode {
 
     [ -f $RF ] && rm -f $RF
 
-    tmpdir=$(mktemp -t -d "test_offline_mode_yamlfilecontent.XXXXXX")
+    tmpdir=$(mktemp -d -t "test_offline_mode_yamlfilecontent.XXXXXX")
 
     # Setup chroot fs and host with test files in different states
     mkdir $tmpdir/tmp
     cp "${srcdir}/${YAML_FILE}" "${tmpdir}/tmp"
 
     cp "${srcdir}/${YAML_FILE}" /tmp
-    sed -i 's/name: instance/name: outstance/' "/tmp/${YAML_FILE}"
+
+    case $(uname) in
```

Comment:
How about adding `xsed` function to the `test_common.sh` and using `sed` or `gsed` there based on `uname`?